src/web-index/webIndex.ts

```diff
@@ -120,10 +117,31 @@ export function createWebIndex(ctx: Context): IWebIndexInterface {
         fileContents = pluginResponse.fileContents;
       }
 
+      fileContents = fileContents.replace(/\$import\('(.+?)'\)/g, (_, relPath: string) => {
+        const result = resolveCSSResource(relPath, {
+          contents: '',
+          ctx,
+          filePath: opts.templatePath,
+          options: ctx.config.stylesheet,
+        });
+        if (result) {
+          return result.publicPath;
+        } else {
+          ctx.log.warn(`Unable to resolve ${result.original}`);
```

Comment:
Will this output the full path that was tested?  If not, please add it.  It's a lot easier to tell what when wrong when you know the absolute path that was hit.

Reply:
`result.original` does output the absolute path.

Reply:
thank you!  It helps a lot with telling users what they did wrong.